test/Unit/Utility/TypeExtensionsTests.cs

```diff
@@ -299,8 +299,8 @@ public void TestTypedDataToObjectJson2()
 
             var input = new TypedData { Json = data };
             var actual = input.ToObject();
-            Assert.IsType<Hashtable>(actual);
-            var actualHash = (Hashtable)actual;
+            Assert.IsType<OrderedHashtable>(actual);
```

Comment:
Should we add a unit test to ensure case-insensitivity for the resulting `Hashtable` of `ToObject(isDurableClient: true)`?

Reply:
In a way, the DurableClient unit tests already cover this case :) . But yes, a small test on this method would be a good reassurance.

Reply:
New test case added
https://github.com/Azure/azure-functions-powershell-worker/blob/5ca8aeb1ee8a1b0e1dafa0aa11ba873cd3d530cd/test/Unit/Utility/TypeExtensionsTests.cs#L363-L381